kyo-core/shared/src/main/scala/kyo/layers.scala

```diff
@@ -0,0 +1,84 @@
+package kyo
+
+import Layers.internal.*
+import kyo.Flat.unsafe
+import kyo.core.*
+import scala.annotation.targetName
+
+sealed trait Layer[-In, +Out, -S]:
+    self =>
+
+    def >>>[In2, Out2, S2](that: Layer[Out & In2, Out2, S2]): Layer[In & In2, Out2, S & S2]       = To(self, that)
```

Comment:
can we have readable names for these instead?

Reply:
Sure! Any suggestions?

Reply:
Maybe to/and/then?

Reply:
`to/and/using`

Reply:
Isn’t using a keyword? :P

Reply:
It compiles 😂